.github/workflows/publish_docker.yml

```diff
@@ -4,15 +4,20 @@ on:
   workflow_dispatch:
     inputs:
       vm_id:
-        description: 'The ID of the VM (binary dst in Docker image)'
-        default: ''
+        description: "The ID of the VM (binary dst in Docker image)"
+        default: ""
```

Comment:
Side note: we should add a yml linter to have consistent yml formatting (I suppose this change is auto-formatting)

Reply:
sounds good